File: numero_primo.py
primo = int (input("ingresa un numero: "))
if primo < 1:
    print("El numero no es primo")
else:
    for i in range(2, primo):
        if primo % i == 0:
            print("El numero no es primo")
            break
    else:
        print("El numero es primo")

File: numeroPrimo.js
let num = parseInt(prompt("Ingresa un número:")); 
let esPrimo = true; 

for (let i = 2; i < num; i++) {
    if (num % i === 0) {
        esPrimo = false; 
        break; 
    }
}

if (esPrimo) {
    console.log(num + " es un número primo."); 
} else {
    console.log(num + " no es un número primo.");
}
